### ISSUE TITLE
Web process crashes with javascript error on reddit.com

### ISSUE THREAD

**Issue by cowile** [atlas-engineer/nyxt#1765]
````
cwl@x1 ~$ nyxt -I https://reddit.com
<INFO> [04:06:41] Listening to socket "/run/user/1000/nyxt/nyxt.socket".
Nyxt version 2.1.1
<INFO> [04:06:41] Loading Lisp file "/home/cwl/.config/nyxt/auto-config.lisp".
<INFO> [04:06:44] Loading history of 48 URLs from "/home/cwl/.local/share/nyxt/history/default.lisp".
<INFO> [04:06:44] Loading "https://reddit.com/".
<WARN> [04:06:47] Warning: Web process crashed for buffer 2
<WARN> [04:06:47] Warning: JavaScript error: GError: Domain: "WebKitJavascriptError", Code: 699, Message: 
<INFO> [04:06:50] Pressed keys: C-x
<INFO> [04:06:50] Deleting socket "/run/user/1000/nyxt/nyxt.socket".
````

**Comment by Ambrevar** [comment]
Can anyone reproduce this?

@cowile Can you clear `/home/cwl/.local/share/nyxt/history/default.lisp` and try again?

**Comment by jmercouris** [comment]
I cannot reproduce. Reddit.com does not crash my browser, neither on old, nor on new Reddit.

**Comment by cowile** [comment]
> @cowile Can you clear `/home/cwl/.local/share/nyxt/history/default.lisp` and try again?

Deleting that file resolved the issue.

**Comment by Ambrevar** [comment]
Hard to say what happened without debugging your session.
My guess is that you got hit by a history race condition.  This should
be fix in the upcoming 2.2.0.
